src/dotnet/commands/dotnet-compile-native/ArgumentsParser.cs

```diff
@@ -1,154 +1,124 @@
 ﻿using System;
 using System.Collections.Generic;
-using System.CommandLine;
 using System.Linq;
+using Microsoft.DotNet.Cli.CommandLine;
+using Microsoft.DotNet.Cli.Utils;
 
```

Comment:
>  are you planning to make a nupkg similar to the Utils package, so when we move dotnet-compile-native to corert, we can consume this there?

We can decide if we are going to do that later. But my gut is telling me the answer will be "no". When `dotnet-compile-native` moves to a different repo, these files will need to be forked.

The right path forward here is to make a real `System.CommandLine` or `Microsoft.Extensions.CommandLine` API which can be used by any command line utility. This just hasn't gotten funded up unto this point. Maybe your need will drive funding a real set of CommandLine APIs post-RC2.